### ISSUE TITLE
sslcapath bug

### ISSUE THREAD

**Issue by laysdra7265** [hackmdio/hackmd#634]
I might encounter a bug with ssl config.
The config in document said that it was an array 
`sslcapath:['./cert/COMODORSAAddTrustCA.crt']`
The problem is `lib/config/index.js:98:25`it try to join it as a path and throw a error.
```console
TypeError: Path must be a string. Received [ 'ssl/ca_bundle.crt' ]
    at assertPath (path.js:7:11)
    at Object.join (path.js:1218:7)
    at Object.<anonymous> (/home/hackmd/lib/config/index.js:98:25)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)

```
If I try to set sslcapath as string path,than app.js can't handle it correctly.
``` console
2017-11-28T11:40:04.258Z - error: uncaughtException: EISDIR: illegal operation on a directory, read date=Tue Nov 28 2017 19:40:04 GMT+0800 (CST), pid=3311, uid=0, gid=0, cwd=/home/hackmd, execPath=/usr/bin/node, version=v6.11.3, argv=[/usr/bin/node, /home/hackmd/app.js], rss=92569600, heapTotal=76632064, heapUsed=54666920, external=1544590, loadavg=[0.0029296875, 0.0302734375, 0.07177734375], uptime=1034304, trace=[column=null, file=null, function=Error, line=null, method=null, native=true, column=19, file=fs.js, function=Object.fs.readSync, line=733, method=readSync, native=false, column=20, file=fs.js, function=tryReadSync, line=488, method=null, native=false, column=19, file=fs.js, function=Object.fs.readFileSync, line=536, method=readFileSync, native=false, column=23, file=/home/hackmd/app.js, function=null, line=53, method=null, native=false, column=5, file=/home/hackmd/app.js, function=null, line=56, method=null, native=false, column=32, file=module.js, function=Module._compile, line=570, method=_compile, native=false, column=10, file=module.js, function=Module._extensions..js, line=579, method=.js, native=false, column=32, file=module.js, function=Module.load, line=487, method=load, native=false, column=12, file=module.js, function=tryModuleLoad, line=446, method=null, native=false, column=3, file=module.js, function=Module._load, line=438, method=_load, native=false, column=10, file=module.js, function=Module.runMain, line=604, method=runMain, native=false, column=7, file=bootstrap_node.js, function=run, line=389, method=null, native=false, column=9, file=bootstrap_node.js, function=startup, line=149, method=null, native=false, column=3, file=bootstrap_node.js, function=null, line=502, method=null, native=false], stack=[Error: EISDIR: illegal operation on a directory, read,     at Error (native),     at Object.fs.readSync (fs.js:733:19),     at tryReadSync (fs.js:488:20),     at Object.fs.readFileSync (fs.js:536:19),     at /home/hackmd/app.js:53:23,     at Object.<anonymous> (/home/hackmd/app.js:56:5),     at Module._compile (module.js:570:32),     at Object.Module._extensions..js (module.js:579:10),     at Module.load (module.js:487:32),     at tryModuleLoad (module.js:446:12),     at Function.Module._load (module.js:438:3),     at Module.runMain (module.js:604:10),     at run (bootstrap_node.js:389:7),     at startup (bootstrap_node.js:149:9),     at bootstrap_node.js:502:3]
```
It looks like some kind of accident commit confused the sslcapath's data type.
I guess it should be an array instead of string value as document said.
The sslcapath's default value have same problem ,it is not an array.

**Comment by SISheogorath** [comment]
Yes, that's right, we should move this:

https://github.com/hackmdio/hackmd/blob/c794412714cdd7b0f6ebfadaf108f72098004d64/app.js#L49-L56

into the index.js of config and handle all these things there. 

If you want to do this feel free to provide a PR :)

**Comment by laysdra7265** [comment]
Sorry, I mixed up two pull requests.
It looks like not a right way to pull request.
BTW,the `protocolusessl` in config won't auto turn on as document said.

**Comment by SISheogorath** [comment]
Fixed by #636

**Comment by kennhung** [comment]
I'm having the same issue on `app.js` line 49 now.

This is the error message, can anyone help me??

```
error:         uncaughtException: EISDIR: illegal operation on a directory, read
Error: EISDIR: illegal operation on a directory, read
    at Object.readSync (fs.js:517:3)
    at tryReadSync (fs.js:342:20)
    at Object.readFileSync (fs.js:379:19)
    at Object.<anonymous> (~/server/app.js:49:17)
    at Module._compile (internal/modules/cjs/loader.js:936:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:947:10)
    at Module.load (internal/modules/cjs/loader.js:790:32)
    at Function.Module._load (internal/modules/cjs/loader.js:703:12)
    at Function.Module.runMain (internal/modules/cjs/loader.js:999:10)
    at internal/main/run_main_module.js:17:11
```

I'm using `npm v6.4.1`, `node v12.10.0`
